scipy/sparse/linalg/eigen/arpack/arpack.py

```diff
@@ -1420,33 +1420,48 @@ def eigsh(A, k=6, M=None, sigma=None, which='LM', v0=None,
         If A is a complex hermitian matrix, 'BE' is invalid.
         Which `k` eigenvectors and eigenvalues to find:
 
-            'LM' : Largest (in magnitude) eigenvalues
+            'LM' : Largest (in magnitude) eigenvalues.
 
-            'SM' : Smallest (in magnitude) eigenvalues
+            'SM' : Smallest (in magnitude) eigenvalues.
 
-            'LA' : Largest (algebraic) eigenvalues
+            'LA' : Largest (algebraic) eigenvalues.
 
-            'SA' : Smallest (algebraic) eigenvalues
+            'SA' : Smallest (algebraic) eigenvalues.
 
-            'BE' : Half (k/2) from each end of the spectrum
+            'BE' : Half (k/2) from each end of the spectrum.
 
         When k is odd, return one more (k/2+1) from the high end.
         When sigma != None, 'which' refers to the shifted eigenvalues ``w'[i]``
         (see discussion in 'sigma', above).  ARPACK is generally better
         at finding large values than small values.  If small eigenvalues are
         desired, consider using shift-invert mode for better performance.
     maxiter : int, optional
-        Maximum number of Arnoldi update iterations allowed
+        Maximum number of Arnoldi update iterations allowed.
         Default: ``n*10``
     tol : float
         Relative accuracy for eigenvalues (stopping criterion).
         The default value of 0 implies machine precision.
     Minv : N x N matrix, array, sparse matrix, or LinearOperator
-        See notes in M, above
+        See notes in M, above.
     OPinv : N x N matrix, array, sparse matrix, or LinearOperator
         See notes in sigma, above.
     return_eigenvectors : bool
-        Return eigenvectors (True) in addition to eigenvalues
+        Return eigenvectors (True) in addition to eigenvalues. This value determines
+        the order in which eigenvalues are sorted. The sort order is also dependent on the which variable.
```

Comment:
could you make which => `which`?

Reply:
I fixed your request.
Changed general word to marked as a variable